samples/subsys/power/stm32_pm/lptim/sample.yaml

```diff
@@ -0,0 +1,14 @@
+sample:
+  name: Low Power Timer Sample for nucleo board
+tests:
+  sample.power.pm.stm32_lptim.lsi:
+    platform_whitelist: nucleo_l4r5zi
```

Comment:
Any reason this is limited to just nucleo_l4r5zi?

Reply:
This is the board I have tested on
I am adding the disco_l475_iot1